src/integration/tests/firstTest.test.tsx

```diff
@@ -76,123 +150,498 @@ describe("First test", () => {
+    const valueInput = inputArea.getByTestId(INPUT_VALUE_ITEM_ID);
+    const buttonAddItem = inputArea.getByTestId(BUTTON_ADD_ITEM_ID);
+
+    const now = new Date(Date.now());
+    const day = now.getDate().toString().padStart(2, "0");
+    const month = (now.getMonth() + 1).toString().padStart(2, "0");
+    const year = now.getFullYear();
+
+    const currentDateShort = `${year}-${month}-${day}`;
+    const typeCateg = "Salário";
+    const msgTitle = "";
+    const valorInput = "100";
+
+    userEvent.type(dataInputArea, currentDateShort);
+    userEvent.selectOptions(categInput, typeCateg);
+    userEvent.type(titleInput, msgTitle);
+    userEvent.type(valueInput, valorInput);
+    userEvent.click(buttonAddItem);
+
+    expect(alertSpy).toHaveBeenCalledTimes(1);
+    const titleInvalid = "Título vazio!";
 
+    expect(alertSpy).toHaveBeenCalledWith(titleInvalid);
+    alertSpy.mockRestore();
   });
 
-  it("should be possible to view the report with columns for Data, Categoria, Titulo, and Valor.", () => {
-    const table = within(screen.getByTestId(TABLE_AREA_ID));
+  it("should must not be possible to insert an item with value equal to zero", () => {
+    const inputArea = within(screen.getByTestId(INPUT_AREA_ID));
+
+    const alertSpy = jest.spyOn(window, "alert").mockImplementation(() => {});
+
+    const dataInputArea = inputArea.getByTestId(INPUT_DATA_ITEM_ID);
+    const categInput = inputArea.getByTestId(INPUT_CATEG_ITEM_ID);
+    const titleInput = inputArea.getByTestId(INPUT_TITLE_ITEM_ID);
+    const valueInput = inputArea.getByTestId(INPUT_VALUE_ITEM_ID);
+    const buttonAddItem = inputArea.getByTestId(BUTTON_ADD_ITEM_ID);
 
-    const data = table.getByText(INPUT_TEXT_DATA);
-    const categ = table.getByText(INPUT_TEXT_CATEGORIA);
-    const title = table.getByText(INPUT_TEXT_TITULO);
-    const valor = table.getByText(INPUT_TEXT_VALOR);
+    const now = new Date(Date.now());
+    const day = now.getDate().toString().padStart(2, "0");
+    const month = (now.getMonth() + 1).toString().padStart(2, "0");
+    const year = now.getFullYear();
+
+    const currentDateShort = `${year}-${month}-${day}`;
+    const typeCateg = "Salário";
+    const msgTitle = "Encora";
+    const valorInput = "0";
+
+    userEvent.type(dataInputArea, currentDateShort);
+    userEvent.selectOptions(categInput, typeCateg);
+    userEvent.type(titleInput, msgTitle);
+    userEvent.type(valueInput, valorInput);
+    userEvent.click(buttonAddItem);
 
-    expect(data).toBeInTheDocument();
-    expect(categ).toBeInTheDocument();
-    expect(title).toBeInTheDocument();
-    expect(valor).toBeInTheDocument();
+    expect(alertSpy).toHaveBeenCalledTimes(1);
+    const valueInvalid = "Valor inválido!";
+    expect(alertSpy).toHaveBeenCalledWith(valueInvalid);
+
+    alertSpy.mockRestore();
   });
 
-  it("should be able to insert an expense item.", () => {});
+  it("should must not be possible to insert an item with value empty", () => {
```

Comment:
Chose "should" or "must"
```suggestion
  it("should not be possible to insert an item with an empty value", () => {
```